08-rails/how-to-use-postgres.md

```diff
@@ -0,0 +1,106 @@
+
+# Setting Up A Rails Project with Postgres
+
+Postgres (or PostgreSQL) is an open-source database which can provide scalability to your application.  By default Rails development sites use sqlite3 which can work fine, but it can be easier to use the same database application in development & production.  
+
```

Comment:
Is it worth talking about what the differences are between PostgreSQL and SQLite, and why you would want to run one over the other?